lib/tasks/pipeline_monitor.rake

```diff
@@ -53,30 +53,39 @@ class CheckPipelineRuns
   end
 
   def self.autoscaling_update(autoscaling_state, t_now)
+    return if Rails.env == "development"
     unless autoscaling_state
       autoscaling_state = {
         t_last: t_now - forced_update_interval,
-        job_count: nil
+        chunk_counts: nil
       }
     end
-    last_job_count = autoscaling_state[:job_count]
-    runs = PipelineRun.in_progress_at_stage_1_or_2
-    runs = runs.where("id > 10") if Rails.env == "development"
-    new_job_count = runs.count
-    if last_job_count.nil?
-      Rails.logger.info("Autoscaling update to #{new_job_count}.")
+    last_chunk_counts = autoscaling_state[:chunk_counts]
+    new_chunk_counts = PipelineRun.count_alignment_chunks_in_progress
+    if last_chunk_counts.nil?
+      Rails.logger.info("Autoscaling update to #{new_chunk_counts}.")
```

Comment:
I would add " chunks" to the print message both here and the line below.   :P)

Reply:
Yes!